ovn-tester/cms/ovn_kubernetes/__init__.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 Canonical
```

Comment:
Same comment as in https://github.com/ovn-org/ovn-heater/pull/173/commits/c7081ee6e73c58e3f7042fdb0dd61f99c7fc4bb8#r1331524518